### ISSUE TITLE
Config file '/docs/mkdocs.yml' does not exist.

### ISSUE THREAD

**Issue by bashirs-investing** [squidfunk/mkdocs-material#1076]
## Description

i pull the latest squidfunk/mkdocs-material image to use on my docker
docker pull squidfunk/mkdocs-material

and when i run the docker : 
 C:\Users\bashirs\GIT> docker run --rm -it -p 8000:8000 -v C:\Users\bashirs\GIT\Docs:/docs squidfunk/mkdocs-material

## i get this error : 

INFO    -  Building documentation... 
Config file '/docs/mkdocs.yml' does not exist.

### Expected behavior
it worked before pulling the latest version 

### System information

i'm running docker desktop  Version 2.0.0.3  on Windows 10

**Comment by squidfunk** [comment]
I just tried the latest Docker image on OS X and it works. I have no way to test it on Windows. Please provide your `mkdocs.yml` and all other data listed in the issue template or we might not be able to help you.

**Comment by bashirs-investing** [comment]
for your question the answer is yes
the mkdocs.yml located in C:\Users\bashirs\GIT\Docs


and this is the content of the mkdocs.yml :


site_name: 'Investing.com Technologies Docs'
site_author: 'Bs'

theme:
  name: 'material'
  language: 'en'
  palette:
    primary: 'indigo'
    accent: 'indigo'
  font:
    text: 'Roboto'
    code: 'Roboto Mono'

**Comment by squidfunk** [comment]
Configuration looks fine to me (assuming the indentation is correct). Did you upgrade Docker lately? `2.0.x` looks like a new major release, which may experience the sudden breakage.

Somebody else experiencing issues with Docker on Windows?

**Comment by bashirs-investing** [comment]
yes i did upgrade the docker . 
but as much as i remember i did run the old  mkdocs  image on the new docker version and it worked

**Comment by squidfunk** [comment]
Could you please verify this again? Just take an older version, for example 4.1.1 or 4.0.0, and try to run it. If it also does not run, it's definitely related to the new Docker version.

**Comment by squidfunk** [comment]
Closing for housekeeping.

**Comment by Genu76** [comment]
I had same issue. 
Reset credentials of Docker for Windows solved it. 
As usual ¯\_(ツ)_/¯

**Comment by FabRons** [comment]
Same issue on CentOS, update docker solved it.

`docker version
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Version:       1.0.5
Go Version:    go1.11.6
OS/Arch:       linux/amd64`

Remove old version:
`dnf remove docker`

Install docker-ce:
`dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo`
`dnf install containerd.io --nobest docker-ce`

Enable docker:
`systemctl enable --now docker`

Open ports:
`firewall-cmd --permanent --add-port 8000/tcp`

Restart firewall:
`systemctl restart firewalld`

Launch mkdocs-material again

**Comment by opus131** [comment]
Just in case anyone else finds this after encountering this issue: 

It is not related to `mkdocs-material`, but rather to the issue described here: https://github.com/docker/toolbox/issues/673

TLDR: 

A possible solution (or rather workaround) is to prepend the command with `MSYS_NO_PATHCONV=1`, so that the full command is 
`MSYS_NO_PATHCONV=1 docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material`

To avoid having to prepend every time, you can add `MSYS_NO_PATHCONV=1` to your `.bashrc`. If that feels to heavy-handed, or causes problems in other places, you can scope it to the docker command by doing smth like this (also in your `.bashrc`):

```
docker()
{
  export MSYS_NO_PATHCONV=1
  "docker.exe" "$@"
  export MSYS_NO_PATHCONV=0
}
```
